mono/mini/mini-runtime.c

```diff
@@ -3674,6 +3674,8 @@ mini_parse_debug_option (const char *option)
 		mini_debug_options.llvm_disable_self_init = TRUE;
 	else if (!strcmp (option, "llvm-disable-inlining"))
 		mini_debug_options.llvm_disable_inlining = TRUE;
+	else if (!strcmp (option, "llvm-disable-implicit-null-checks"))
```

Comment:
Please update mono man page to include the new option

Reply:
Done!